Resources/Prototypes/_NF/Entities/Objects/base_contraband.yml

```
@@ -129,6 +129,16 @@
   - type: Contraband
     turnInValues: *oneFUC
 
+- type: entity
+  parent: BaseC3PirateContrabandNoValue
+  id: BaseC3PirateContrabandHighValue
```

```suggestion
  id: BaseC3PirateContrabandHighValue # Mono
```
Ideally, this should be in Monolith namespace.